src/components/wallet/methods/trustAsset.ts

```diff
@@ -19,10 +19,11 @@ export default async function trustAsset(
   this.error = null
   console.log(this.loading)
   const finish = () => (this.loading = { ...this.loading, trust: false })
+  let inputs
   try {
     if (!asset || !issuer) {
-      let instructions = await this.setPrompt({ message: '{Asset} {Issuer}' })
-      ;[asset, issuer] = instructions.split(' ')
+      inputs = await this.setPrompt({ message: '{Asset} {Issuer}' })
+      ;[asset, issuer] = inputs[0].value.split(' ')
```

Comment:
Is there a way to leverage the new multi-input prompt here instead of adopting the split(' ') logic from before?

Reply:
Yes, thats my next step PR haha